recipe/post-link.bat

```diff
@@ -1,21 +1,53 @@
-@ECHO OFF
+SETLOCAL ENABLEDELAYEDEXPANSION
 
 REM The menuinst v2 json file is not compatible with menuinst versions
-REM older than 2.1.0. Copy the appropriate file as the menu file.
+REM older than 2.1.1. Copy the appropriate file as the menu file.
 
 SET LOGFILE=%PREFIX%\.messages.txt
+SET MENU_DIR=%PREFIX%\Menu
+SET MENU_PATH=%MENU_DIR%\%PKG_NAME%_menu.json
 
-REM Cannot use usual FOR-loop way to assign output of command because
-REM prefixes with spaces in the name cannot be properly escaped.
-"%CONDA_PYTHON_EXE%" -c "import menuinst, sys; sys.exit(1 if tuple(int(x) for x in menuinst.__version__.split(\".\"))[:3] < (2, 1, 0) else 0)"
+IF EXIST "%CONDA_PYTHON_EXE%" (
+    SET PYTHON_CMD="%CONDA_PYTHON_EXE%"
+    GOTO :get_menuinst
+)
+
+IF EXIST "%PREFIX%\_conda.exe" (
+    SET PYTHON_CMD="%PREFIX%\_conda.exe" python
+    GOTO :get_menuinst
+)
+
+GOTO :menuinst_too_old
+
+:get_menuinst
+%PYTHON_CMD% -c "import menuinst, sys; sys.exit(1 if tuple(int(x) for x in menuinst.__version__.split('.'))[:3] < (2, 1, 1) else 0)"
```

Comment:
There is `menuinst >=2.1.0` in `run_constrained`. Should it be there `menuinst >=2.1.1`?